src/main/java/com/sda/quiz/initialization/Initializer.java

```diff
@@ -0,0 +1,69 @@
+package com.sda.quiz.initialization;
+
+import com.sda.quiz.domain.Question;
+import com.sda.quiz.domain.QuestionCategory;
```

Comment:
CommandLiner to jest bean tylko do testow wpisywania do bazy danych, sprawdz czy do bazy h2 wpisuja sie utworzone przez Ciebie rzeczy - jezeli tak to bean mozesz usunac i przejsc do pisania klas RestController

Reply:
To zadanie miało tworzyć wpisy w bazie, RestController będzie dostarczony później w innym branchu.
Zostawmy tworzenie bean bo w konfiguracji mamy create drop.